no more Meteor.absoluteUrl()

jorisroling opened the pull request:
In order to make this package work on different port numbers (when developing localy), AND from different URL's in production, I propose this change:

It makes no mention of the protocol, host or port number, when creating a **orionFiles** and as such the images are accessible from the same base as the web page itself. Exactly what is needed.

vsivsi commented:
Hi, thanks for the PR!  

I'm actually no longer using Orion (or rather, never really did except as an experiment...)

So I don't have the experience with Orion (or the incentive) to keep this package up. If you are using it actively and interested, I'd be happy to transfer it over to you to maintain.

Otherwise, I'm afraid it's a bit of an orphan.

jorisroling commented:
Yes, please, I'm an active Orion user, and I have developed some Orion packages.
If you want, I'm happy to maintain it.

Regards,
Joris

> On 20 Oct 2015, at 19:07, Vaughn Iverson notifications@github.com wrote:
> 
> Hi, thanks for the PR!
> 
> I'm actually no longer using Orion (or rather, never really did except as an experiment...)
> 
> So I don't have the experience with Orion (or the incentive) to keep this package up. If you are using it actively and interested, I'd be happy to transfer it over to you to maintain.
> 
> Otherwise, I'm afraid it's a bit of an orphan.
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/vsivsi/orion-file-collection/pull/1#issuecomment-149635132.

vsivsi commented:
Okay, I just added you as a collaborator.  That should give you push access for now.  Looks like I can't transfer the repo to you while you have a fork of the repo in your account.

vsivsi commented:
I'll need to look into what it takes to transfer a project within Atmosphere.  Although nothing prevents you from just creating `jorisroling:orion-file-collection` I suppose...

jorisroling commented:
I have removed the fork. Could you retry.

> On 20 Oct 2015, at 20:41, Vaughn Iverson notifications@github.com wrote:
> 
> Okay, I just added you as a collaborator. That should give you push access for now. Looks like I can't transfer the repo to you while you have a fork of the repo in your account.
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/vsivsi/orion-file-collection/pull/1#issuecomment-149660889.

jorisroling commented:
Thanks, I might do that.

> On 20 Oct 2015, at 20:42, Vaughn Iverson notifications@github.com wrote:
> 
> I'll need to look into what it takes to transfer a project within Atmosphere. Although nothing prevents you from just creating jorisroling:orion-file-collection I suppose...
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/vsivsi/orion-file-collection/pull/1#issuecomment-149661734.

vsivsi commented:
I just added you as a maintainer on `vsivsi:orion-file-collection` on Atmosphere.  So you should be able to publish new versions.

jorisroling commented:
Great! Thanks...

J*

> On 20 Oct 2015, at 22:16, Vaughn Iverson notifications@github.com wrote:
> 
> I just added you as a maintainer on vsivsi:orion-file-collection on Atmosphere. So you should be able to publish new versions.
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/vsivsi/orion-file-collection/pull/1#issuecomment-149690037.

jorisroling commented:
And it all works, Github & Atmosphere, again... thanks!

> On 20 Oct 2015, at 22:16, Vaughn Iverson notifications@github.com wrote:
> 
> I just added you as a maintainer on vsivsi:orion-file-collection on Atmosphere. So you should be able to publish new versions.
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/vsivsi/orion-file-collection/pull/1#issuecomment-149690037.

vsivsi commented:
Great!  In the next release you might want to bump the version of `vsivsi:file-collection` to the latest (1.2.0)  Also, I'm in the habit of tagging releases on github when I publish to Atmosphere.

jorisroling commented:
Sure, will do...

> On 20 Oct 2015, at 22:30, Vaughn Iverson notifications@github.com wrote:
> 
> Great! In the next release you might want to bump the version of vsivsi:file-collection to the latest (1.2.0) Also, I'm in the habit of tagging releases on github when I publish to Atmosphere.
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/vsivsi/orion-file-collection/pull/1#issuecomment-149693000.
